### ISSUE TITLE
Chrome unregisters MSW after 5 mins of inactivity / idle on MSW Examples Repo

### ISSUE THREAD

**Issue by jkinger** [kettanaito/msw#1744]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Browsers

Chromium (Chrome, Brave, etc.)

### Reproduction repository

https://github.com/mswjs/examples/tree/master/examples/rest-react

### Reproduction steps

1. Follow the "Getting Started" guide in the Readme of https://github.com/mswjs/examples/tree/master/examples/rest-react
2. Run `yarn start`
3. Using Chrome, open the localhost URL.
4. Once opened, leave the browser tab open in the background without any user activity for at least 5 minutes and ensure DevTools is not open (this is important).
5. Come back after 5 minutes and try to hit the submit button.  

### Current behavior

Nothing happens when the submit button is pressed because Chrome unregistered the MSW from inactivity.

### Expected behavior

With the KeepAlive addition ([Issue  367](https://github.com/mswjs/msw/issues/367)), I would assume the submit button would still work correctly even after an elapsed time of inactivity passes.

**Comment by kettanaito** [comment]
Hi, @jkinger. Thanks for submitting this. 

We have a ping/pong messaging in place to prevent this behavior. I wonder if that messaging happens in your case or if it's related to something else. 

Can you please confirm your version of `msw` by running `npm ls msw` in your project?

**Comment by jkinger** [comment]
When I ran `npm ls msw` it gave the following:
```
└─┬ rest-react-example@0.0.0 -> ./examples/rest-react
  └── msw@0.20.3 invalid: "latest" from examples/rest-react
  ```
  
I fixed the package to be `"msw": "^1.3.1"` and confirmed with `npm ls msw` (below):
```
└─┬ rest-react-example@0.0.0 -> ./examples/rest-react
  └── msw@1.3.1
  ```
  
Re-ran `npx msw init public` and confirmed that `/examples/rest-react/public/mockServiceWorker.js` was also reflecting the new version as well.

I then ran through the above "Reproduction steps" again and was unable to reproduce now so closing.

If any takeaways can be gathered from this it might be the [MSW Example repo](https://github.com/mswjs/examples/tree/master) could use some updating but I'm sure there are bigger fish to fry.

Thanks all!

**Comment by kettanaito** [comment]
@jkinger, I'm glad to hear you've resolved it! The `latest` tag can get confusing with pre-1.0 releases, I know. 

> If any takeaways can be gathered from this it might be the [MSW Example repo](https://github.com/mswjs/examples/tree/master) could use some updating but I'm sure there are bigger fish to fry.

That's true! In fact, we already have an updated version of examples at https://github.com/mswjs/examples-new. It's going to replace the current examples for MSW 1.0 in the near future. 

Keep on building!
